criteria.md

```diff
@@ -0,0 +1,47 @@
+### Maturity Level 1 (Sandbox)
+ 
+| Category | Criteria Statement |
+|---|---|
+| Access Control | The project's version control system MUST require multi-factor authentication (MFA) for users when modifying the project repository settings or accessing sensitive data. |
+| Access Control | The project's version control system MUST restrict collaborator permissions to the lowest available privileges by default. |
+| Access Control | The project's version control system MUST prevent accidental direct commits against the primary branch. |
+| Access Control | The project's version control system MUST prevent accidental deletion of the default branch. |
+| Build & Release | The project's build and release pipelines MUST NOT execute arbitrary code that is input from outside of the build script. |
+| Build & Release | All software releases MUST be assigned a standardized unique version identifier. |
+| Build & Release | Any websites related to the project development and release MUST be delivered using SSH or HTTPS and TLS. |
```

Comment:
HTTPS implies TLS (I think always, but at least does in practice), so I don't think there is a reason to write it this way.
```suggestion
| Build & Release | Any websites related to the project development and release MUST be delivered using SSH or HTTPS. |
```